src/noc/containers/noc-field/field-type-enum.model.ts

```diff
@@ -0,0 +1,9 @@
+export type FieldTypeEnum =
```

Comment:
This is not an enum, which makes the name confusing.

Reply:
redundant code will remove it